battleship.py

```diff
@@ -14,38 +14,35 @@ def retrieve_size():
 
 def clear_grid():
     new_grid = []
-    for i in range(16):
+    for i in range(grid_size):
         new_grid.append([])
     for i, k in enumerate(new_grid):
-        for a in range(16):
+        for a in range(grid_size):
             new_grid[i].append("0")
     return new_grid
-    # TODO: Variable grid size
 
 
 def place_ships():
-    # Temporary static ship placement for development
     ships = {
-        "carrier": [[1, 1], [2, 1], [3, 1], [4, 1], [5, 1]],
-        "cruiser": [[4, 2], [4, 3], [4, 4]]
+        "carrier": [[1, 1], [1, 2], [1, 3], [1, 4], [1, 5]],
+        "cruiser": [[2, 4], [3, 4], [4, 4]]
     }
     return ships
-    # TODO: implement random ship placement
 
 
 def get_target():
     choice = input("Where would you like to aim? (example: '12,3')")
-    if len(choice) > 7:
+    if not (choice.split(','))[0].isdigit() or not (choice.split(','))[1].isdigit() \
+            or 1 <= int((choice.split(','))[0]) > grid_size or 1 <= int((choice.split(','))[1]) > grid_size:
         print("Your coordinate was invalid. Please follow the example and try again.")
         get_target()
     else:
         choice_list = choice.split(',')
         target = []
         for i in choice_list:
             target.append(int(i))
+        target = [target[1], target[0]]
```

Comment:
this works. If you want you can also use python's built in function reverse()

```
target.reverse()
```
You don't even have to store it in a new variable you just call it on the variable like that and it will reverse it.